labs/a1.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Advanced Lab 1
+layout: lab
+---
+
+### A Note on Labs
+Labs are graded on completion. Treat this lab as seeds of exploration instead 
+of just a grade. 
+
+As this is the first lab, it is intentionally simple and easy to complete.  Bash scripting
+isn't a paticular goal of the DeCal but this lab should introduce you to some fun bash
+features you may not have encountered before, such as loops and shell expansions. 
+
+### Workflow
+This lab can be done on your own UNIX-like machine, or you can ssh into
+`tsunami.ocf.berkeley.edu` using your OCF account to finish the lab there. As always,
+`man` and Google will be your friends.
+
+## Question 1
+
+Using Bash functions, write a script that generates a user-specified number of files of
```

Comment:
I'd make this sentence "write a script `mkrandom.sh` that generates...." and then add the `.sh` extension in the example. IMO it's slightly clearer that way. (same with the rest of the questions)